.github/workflows/build-and-test.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
           FBN_CLIENT_SECRET: ${{ secrets.DEVELOP_FBN_CLIENT_SECRET }}
           FBN_LUSID_API_URL: ${{ secrets.DEVELOP_FBN_LUSID_API_URL }}
           FBN_APP_NAME: ${{ secrets.DEVELOP_FBN_CLIENT_ID }}
+          FBN_LUSID_PAT: ${{ secrets.DEVELOP_FBN_ACCESS_TOKEN }}
```

Comment:
i think this should be `FBN_LUSID_ACCESS_TOKEN`

Reply:
Thanks. Updated.